frontend/src/constants/reducerTypes.tsx

```diff
@@ -10,5 +10,7 @@ export const LOOKUP_CODE = 'lookupCode';
 export const GET_LOOKUP_CODES = 'GET_LOOKUP_CODES';
 
 // Auth
+export const ACCESS_REQUEST = 'accessRequest';
```

Comment:
What are these consts for?

Reply:
consistent use throughout the application. This value is needed by the reducer and also by the selector, and action. Using the constants saves me from having to remember the keys these values are saved/retrieved by.